tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -5017,4 +5017,154 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+		$order               = $this->create_order_with_presentment_email_data( $checkout_session_id );
+		$order->set_billing_country( 'FR' ); // France is an EEA country.
+		$order->save();
+
+		ob_start();
+		$this->mock_gateway->add_email_currency_conversion_notice( $order );
+		$output = ob_get_clean();
+
+		WC_Stripe_Database_Cache::delete( 'checkout_session_' . $checkout_session_id );
+
+		$this->assertStringContainsString( 'European Central Bank (ECB) interbank rate', $output );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_email_currency_conversion_notice_appends_ecb_fee_html_for_eea_customer is not named in camelCase. (undefined)

(CamelCaseMethodName)